docs/sources/administration/configuration.md

```diff
@@ -1201,18 +1201,22 @@ Configure Grafana's Jaeger client for distributed tracing.
 You can also use the standard `JAEGER_*` environment variables to configure
 Jaeger. See the table at the end of https://www.jaegertracing.io/docs/1.16/client-features/
 for the full list. Environment variables will override any settings provided here.
+These environment variables follow the `GF_<section>` convention above,
+i.e. `JAEGER_SAMPLER_TYPE` becomes `GF_TRACING_JAEGER_SAMPLER_TYPE`.
 
 ### address
 
 The host:port destination for reporting spans. (ex: `localhost:6831`)
 
-Can be set with the environment variables `JAEGER_AGENT_HOST` and `JAEGER_AGENT_PORT`.
+Can be set with the environment variables `GF_TRACING_JAEGER_AGENT_HOST`
+and `GF_TRACING_JAEGER_AGENT_PORT`.
 
 ### always_included_tag
 
 Comma-separated list of tags to include in all new spans, such as `tag1:value1,tag2:value2`.
 
-Can be set with the environment variable `JAEGER_TAGS` (use `=` instead of `:` with the environment variable).
+Can be set with the environment variable `GF_TRACING_JAEGER_TAGS`
```

Comment:
Information about `JAEGER_TAGS` is still needed, but you can explain that `GF_TRACING_JAEGER_TAGS` can be used as well.